_posts/2024-06-12-release.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Release 2024.24
+author: Priya
+date: 2024-06-12
+category: Jekyll
+layout: post
+---
+### Enhancement:
+
+#### Combine Dataset Feature Improvements:
+Mammoth now provides full visibility of column names without the need for hovering, saving time and improving efficiency. Scrolling has been reduced to easily locate unmatched columns, and the search functionality has been added for quicker identification of specific columns.
+
+#### Onboarding Flow Improvements:
+Mammoth now delivers a streamlined onboarding process, offering seamless options to upload data and connect a database immediately after signup.
```

Comment:
Remove the headings from Combine dataset feature improvement and Onboarding flow improvements. Just capture these as enhancements.
Also for the text of this particular enhancement:
Improvements to onboarding flow for self serve users.